### ISSUE TITLE
Can't edit a wiki page if there is a link to an profile that doesn't exist anymore

### ISSUE THREAD

**Issue by Niclas124** [humhub/wiki#251]
Hi together,

for me and other participants of the hub it wasn't possible to edit a wiki page if there is a link to an profile that doesn't exist anymore. When the page was edit, it was possible to press the button "save" and there was also the message that the changes were saved, but  nothing was saved and the changes didn't exist. When I deleted the old links to the profiles on the page that didn't exist anymore in the hub, it works. 

Maybe this helps other people with the same problem and maybe someone can fix the bug :)
In that case, thank you very much!

Have a nice day :)

**Comment by yurabakhtin** [comment]
@Niclas124 Could you give more details where excatly the profile link is used?
It would be better to see some screenshots or errors what you get and what is expected, thanks.

**Comment by Niclas124** [comment]
Yes of course, sorry for that. It is the first time for me reporting a bug. 

If you can see the screenshot, there is a table and at the last column you can see a linked profile which you can always add with @...  . Unfortunately I can't reproduce the bug, because it isn't possible to create manually a link to a profile that doesn't exist. The problem was that there in the column in some rows were linked profiles that didn't exist anymore. The problem was then, like I described in the beginning post, I edited the page (in the screenshot the button is on the right) and edited any text and when I press the button "Save", there was also like normal the message "Saved", but the text was the same like before editing and any changes weren't save . With deleting the linked profiles, which doesn't exist any more, it works.

I hope I could describe the bug better. If there are still any questions, you can always ask. Thanks!


![Screenshot 2022-11-24 091607](https://user-images.githubusercontent.com/118898265/203729209-727f5809-eebe-42a2-b281-8995bca120d8.jpg)

**Comment by yurabakhtin** [comment]
@Niclas124 Please review my test, and let me know if I didn't test this correctly:

![test_deleted_user](https://user-images.githubusercontent.com/6995524/203774863-fec57782-52bd-43e0-af75-d77f10e48de9.png)

**Comment by Niclas124** [comment]
@yurabakhtin 
You tested it correctly, but in my case there was still the linked profile "@Pass Admin" after deleting it, like @Test Legal, and not something like: "[Pass Admin](mention....." .Cause of that, you wasn't able to edit successfully the page and if you edit the text it wasn't successfully saved.

**Comment by Niclas124** [comment]
And if you clicked on the linked profile, that was deleted, you get on the top of the page.

**Comment by yurabakhtin** [comment]
@Niclas124 As I have understood you cannot reproduce this bug too.
Even if it can be reproduced I am not sure we should fix this somehow because after deleting of a user we cannot find inside each content and replace such profile links to some more correct text without link.

@luke- Do you agree to keep it as is?

**Comment by luke-** [comment]
I would also rather leave the current behavior as it is. Maybe in the future we can replace not retrievable Mentionings with a link to the People page and text (Deleted user) in the Markdown parser.
